includes/forms/class-llms-form-handler.php

```diff
@@ -338,12 +356,11 @@ public function submit( $posted_data, $location, $args = array() ) {
 		 *               Triggered by `do_action_ref_array()` instead of `do_action()` allowing modification
 		 *               of `$posted_data` and `$fields` via hooks.
 		 *
-		 * @param array $posted_data Array of user-submitted data (passed by reference).
-		 * @param string $location Form location.
-		 * @param array[] $fields Array of LifterLMS Form Fields (passed by reference).
-		 * @param array $args Additional arguments from the form retrieval function.
+		 * @param array   $posted_data Array of user-submitted data (passed by reference).
+		 * @param string  $location    Form location.
+		 * @param array[] $fields      Array of LifterLMS Form Fields (passed by reference).
 		 */
-		do_action_ref_array( "lifterlms_before_user_${action}", array( &$posted_data, $location, &$fields, $args ) );
+		do_action_ref_array( "lifterlms_before_user_${action}", array( &$posted_data, $location, &$fields ) );
```

Comment:
Why did the $args arg get removed from these actions/filters?

Reply:
sounds sensible, the argument should be removed from the method in that case too then, right?